cmd/linux.go

```diff
@@ -51,6 +51,8 @@ const (
 	ManjaroARM
 	// Deepin distribution
 	Deepin
+	// Raspbian
```

Comment:
the pattern here is `// Raspbian distribution` otherwise codacy complains..

Reply:
just fixed it! and Codacy did complain!

Reply:
Is there a way to run Codacy checks locally?  I know it does more than just what `golint ./...` does although in this case `golint` would have complained had I run it.